package.json

```diff
@@ -31,7 +31,7 @@
     "build:js": "webpack --mode production",
     "build:type": "tsc --module es6 --declaration --emitDeclarationOnly --outDir build/esm",
     "start": "npm-run-all -p start:css storybook",
-    "start:css": "NODE_ENV=production postcss src/tailwind.css -o .storybook/styles.css --watch",
+    "start:css": "postcss src/tailwind.css -o .storybook/styles.css --watch",
```

Comment:
lets maintain this so to purge?

Reply:
i realise with the `NODE_ENV=production` the --watch not working when developing